widgetssdk/src/main/java/com/glia/widgets/di/Dependencies.java

```diff
@@ -63,11 +63,7 @@ public static void onAppCreate(Application application) {
                 new ApplicationLifecycleManager(),
                 controllerFactory.getChatHeadController()
         );
-        ActivityWatcherForCallVisualizer activityWatcherForCallVisualizer = new ActivityWatcherForCallVisualizer(
-                controllerFactory.getCallVisualizerController(),
-                controllerFactory.getScreenSharingController(),
-                controllerFactory.getDialogController(),
-                controllerFactory.getChatHeadController());
+        ActivityWatcherForCallVisualizer activityWatcherForCallVisualizer = new ActivityWatcherForCallVisualizer(getControllerFactory().getDialogController(), getControllerFactory().getActivityWatcherController());
```

Comment:
NIT. I would suggest moving `getControllerFactory().getDialogController()` to a separate line.

Reply:
done